packages/kubernetes-1.18/kubelet-config

```diff
@@ -20,12 +20,14 @@ clusterDNS:
 - {{settings.kubernetes.cluster-dns-ip}}
 resolvConf: "/etc/resolv.conf"
 hairpinMode: hairpin-veth
+readOnlyPort: 0
 cgroupDriver: systemd
 cgroupRoot: "/"
 runtimeRequestTimeout: 15m
 featureGates:
   RotateKubeletServerCertificate: true
   CSIMigration: false
+protectKernelDefaults: true
```

Comment:
I think @bcressey should comment on the specific settings in this PR, but separate from that, I'm almost positive we'd want to apply them to all Kubernetes versions, not just 1.18.  I don't think we should separate those changes into different PRs; we don't intend for the different versions to have different behaviors.